Geometry_Engine/Create/TransformMatrix.cs

```diff
@@ -197,18 +201,60 @@ public static TransformMatrix OrientationMatrixGlobalToLocal(Cartesian csTo)
 
         /***************************************************/
 
+        [Description("Creates a TransformMatrix corresponding to a orientation of the geometry to Global coordinates, with origin at {0,0,0} from the local system provided.")]
+        [Input("csFrom", "The cartesian coordinate system to orient the geometry from.")]
+        [Output("transform", "The created TransformMatrix.")]
         public static TransformMatrix OrientationMatrixLocalToGlobal(Cartesian csFrom)
         {
             return OrientationMatrixGlobalToLocal(csFrom).Invert();
         }
-    
+
         /***************************************************/
 
+        [Description("Creates a TransformMatrix corresponding to a orientation of the geometry from one local coordinate system to another.")]
+        [Input("csFrom", "The cartesian coordinate system to orient the geometry from.")]
+        [Input("csTo", "The cartesian coordinate system to orient the geometry to.")]
+        [Output("transform", "The created TransformMatrix.")]
         public static TransformMatrix OrientationMatrix(this Cartesian csFrom, Cartesian csTo)
         {
             return OrientationMatrixGlobalToLocal(csTo) * OrientationMatrixLocalToGlobal(csFrom);
         }
 
+        /***************************************************/
+        /**** Random Geometry                           ****/
+        /***************************************************/
+
+        [Description("Creates a random TransformMatrix based on a seed. If no seed is provided, a random one will be generated.")]
+        [Input("seed", "Input seed for random generation. If -1 is provided, a random seed will be generated.")]
+        [Input("minVal", "Minimum value to be used for each position in the created random matrix.")]
```

Comment:
Not sure if that makes sense, so please feel free to simply ignore (applies to all similar cases)
```suggestion
        [Input("minVal", "Minimum value to be used for each property in the created random matrix.")]
```

Reply:
Will change to "element", using nomenclature from here https://en.wikipedia.org/wiki/Matrix_(mathematics)#Definition